lms/static/js/ccx/view/schedule_tree.js

```diff
@@ -0,0 +1,53 @@
+var define = window.define || RequireJS.define;  // jshint ignore:line
+
+define(
+  [
+    'backbone',
+    'jquery',
+    'underscore',
+    'gettext',
+    'js/ccx/model/schedule_model',
+    'js/ccx/view/schedule_tree_node'
+  ],
+  function ( Backbone, $, _, gettext, scheduleModel, scheduleTreeNode) {
+    'use strict';
+    var ccxScheduleTreeView = Backbone.View.extend({
+      render: function () {
+        _.each(this.collection.models, this.processChapters, this);
+        return this;
+      },
+
+      processChapters: function(chapter){
+        var ccxScheduleTreeNode = new scheduleTreeNode.
+          ccxScheduleTreeNodeView({ model: chapter });
+        ccxScheduleTreeNode.render();
+        this.$el.append(ccxScheduleTreeNode.el);
+        _.each(chapter.toJSON().children, this.processSequential, this);
```

Comment:
The `ccxScheduleTreeNodeView` should have the necessary information to render this itself, no? This feels like "digging into collaborators". Consider pushing this into `ccxScheduleTreeNodeView`.